src/app/child-dev-project/notes/notes-manager/notes-manager.component.spec.ts

```diff
@@ -78,16 +79,17 @@ describe("NotesManagerComponent", () => {
     snapshot: { queryParams: {} },
   };
 
-  const testInteractionTypes: InteractionType[] = [
-    {
-      id: "HOME_VISIT",
-      label: "Home Visit",
-    },
-    {
-      id: "GUARDIAN_TALK",
-      label: "Talk with Guardians",
-    },
-  ];
+  const testInteractionTypes: InteractionType[] =
+    EnumOrdering.imposeTotalOrdering([
```

Comment:
Is this required?

Reply:
This (and the two occurrences below) are required because the type of the enum is declared as ordered.